keras2onnx/_builtin.py

```diff
@@ -1879,9 +1879,11 @@ def _prepare_StridedSlice(node, target_opset):
 
         shrink_mask = (shrink_axis_mask >> idx) & 1
         if shrink_mask != 0:
-            shrink_begin = begin_item + _cal_tensor_shape(data_input)[idx] if begin_item < 0 else begin_item
-            new_begin.append(shrink_begin)
-            new_end.append(shrink_begin + 1)
+            new_begin.append(begin_item)
+            if begin_item == -1:
```

Comment:
if this is -1, does it mean the output is inverse?

Reply:
no, shrink mask actually means slice and squeeze out.